File: 08/winter/turning_radius.py
import math
line = raw_input()
while line:
    l, w, theta_in = map(float, line.split())
    r = (l / math.tan(math.radians(theta_in)))
    print "rear inside turn radius: %.2f" % r
    print "front outside wheel angle: %.2f" % math.degrees(math.atan(l/(r+w)))
    print 
    try: line = raw_input()
    except: break

File: 08/winter/solutions/turning_radius.java
import java.util.Scanner;

class turning_radius {

	public static void main(String[] arguments) {

		Scanner in = new Scanner(System.in);

		while (in.hasNextDouble()) {

			double l = in.nextDouble();
			double w = in.nextDouble();
			double theta_in = in.nextDouble() * Math.PI / 180;
			double r = l / Math.tan(theta_in);
			double theta_out = Math.atan(l / (r+w)) * 180 / Math.PI;

			System.out.printf(" rear inside  turn radius: %.2f%n", r);
			System.out.printf("front outside wheel angle: %.2f%n", theta_out);
			System.out.println();
		}
	}
}
